### DISCUSSION TITLE
Can you suggests some video course for nuxt 3?

### DISCUSSION THREAD

**Discussion by realtebo** [nuxt/nuxt#17329] [General]
Hi !

I am an 'old' Vue 1,2,3 developer but I am totally new to Nuxt.

I was waiting for Nuxt 3 and now I think it's time to learn it.

Do you have some video courses (*professional level only*) about nuxt 3 already released to suggest to me?

**Comment by Kingside88** [comment]
I can suggest the video channel of Erik.
Of course you would also find him, if you would search for "nuxt 3" on youtube.
https://www.youtube.com/c/ProgramWithErik/videos

**Reply by realtebo** [reply]
Thanks. I am looking for a more professional course. Paid. because i learn 80% of his video just reading and trying. And I already did both, watched YT serie and learned and tried and read docs and opened source code etc..
